config

```diff
@@ -0,0 +1,3 @@
+PORT=5678
+PATH=./database
```

Comment:
Can you add this directory to the .gitignore file? Otherwise it will be cumbersome to add files to the index.